src/cli/vector_search.rs

```diff
@@ -134,41 +134,34 @@ fn run(
                 let id = rec.get("id").unwrap().as_str().unwrap();
```

Comment:
Won't this line and the line about content both panic on unexpected types?

Reply:
This is inside an if: 
```
if rec.contains("id") && rec.contains("content") {
```